arwenmandos/util.go

```diff
@@ -45,43 +45,30 @@ func convertAccount(testAcct *mj.Account) (*worldmock.Account, error) {
 
 	for _, mandosESDTData := range testAcct.ESDTData {
 		tokenName := mandosESDTData.TokenIdentifier.Value
-		tokenValue := mandosESDTData.Value.Value
-		tokenNonce := mandosESDTData.Nonce.Value
 		isFrozen := mandosESDTData.Frozen.Value > 0
-		tokenKey := worldmock.MakeTokenKey(tokenName, tokenNonce)
-		tokenData := &esdt.ESDigitalToken{
-			Value:      tokenValue,
-			Type:       uint32(core.Fungible),
-			Properties: makeESDTUserMetadataBytes(isFrozen),
-			TokenMetaData: &esdt.MetaData{
-				Name:  tokenName,
-				Nonce: tokenNonce,
-			},
+		for _, instance := range mandosESDTData.Instances {
+			tokenNonce := instance.Nonce.Value
+			tokenKey := worldmock.MakeTokenKey(tokenName, tokenNonce)
+			tokenBalance := instance.Balance.Value
+			tokenData := &esdt.ESDigitalToken{
+				Value:      tokenBalance,
+				Type:       uint32(core.Fungible),
```

Comment:
Why is type always Fungible? Why have non-zero nonce for Fungible ESDT?

Reply:
Not yet properly implemented. Will do

Reply:
... in the next PR